packages/forklift-console-plugin/src/modules/Providers/utils/validators/provider/providerValidator.ts

```diff
@@ -27,7 +27,7 @@ export function providerValidator(
       validationError = ovirtProviderValidator(provider);
       break;
     case 'vsphere':
-      validationError = vsphereProviderValidator(provider, subType, secret?.data?.cacert);
+      validationError = vsphereProviderValidator(provider, subType, secret);
```

Comment:
do we still need to pass the whole secret instead of just the cacert in all relevant places?

Reply:
The cacert isn't read anymore for the validation, just the const insecureSkipVerify field. I could just pass that if you'd like, but I'm not sure it makes much of a difference here.

Reply:
Updated